server/settings.example.yml

```diff
@@ -98,7 +98,8 @@ STARTER_BANK_ITEMS:
   - itemName: IronSword
   - itemName: IronHatchet
   - itemName: IronPickaxe
-
+  - itemName: PacifySpellScroll
+    quantity: 9
```

Comment:
Not needed to be added to these default settings.

Reply:
is there a way to test those items without adding them to this settings?

Reply:
It is ok to add them here to the settings for development, but they just don't need to be a part of the PR.
To note, this is just the example/defaults file, but you can set your own overrides in your own settings file if you make a copy of settings.example.yml called settings.yml in the same directory, and change that however you want. Any changes to that file then will not be tracked in git so you can configure it however you want and your development settings won't be included in the PR.

Reply:
Thank you very much it will help a lot.